addons/mass_mailing/static/src/js/mass_mailing_editor.js

```diff
@@ -281,4 +282,30 @@ if ($("#editable_area").html().indexOf('on_change_model_and_list') !== -1) {
     $("#editable_area").empty();
 }
 
+// Adding compatibility for the outlook compliance of mailings.
+// Commit of such compatibility : a14f89c8663c9cafecb1cc26918055e023ecbe42
+var background_mailing = options.registry.background.extend({
+    background: function(type, value, $li) {
+        this._super(type, value, $li);
+        var $table_target = this.$target.find('table:first');
+        if ($table_target) {
+            if (value && value.length) {
+                // style should be inline since it is an email
+                var background_style = 'background-image: url(' + value + ');\
+                                        background-size: cover;\
+                                        background-position: center center;'
+
+                var origin_style = (this.$target.attr("style") || '').replace(/background-image:[^;]+/, '');
+                $table_target.attr("style", background_style + origin_style);
+                $table_target.addClass("oe_img_bg");
+            } else {
+                $table_target.css("background-image", "");
+                $table_target.removeClass("oe_img_bg").removeClass("oe_custom_bg");
+            }
+        }
+    }
+});
```

Comment:
- Duplication of code
- Inline style is not to be put because of mail edition, this is automatically done when saving
- Unnecessary need for XML update

If I understand the problem right, the fix should only be an `include` of the `background` option for the mass mailing editor which changes the `$target` to the right element on `start`. This could be done with something like this: https://github.com/odoo/odoo/blob/10.0/addons/website/static/src/js/website.snippets.editor.js#L228 (in master there is a `setTarget` method).